website/photos/tasks.py

```diff
@@ -22,46 +22,49 @@ def process_album_upload(
     archive_upload_id: str, album_id: int, uploader_id: int | None = None
 ):
     upload = TemporaryUpload.objects.get(upload_id=archive_upload_id)
-    archive = upload.file
 
     try:
         album = Album.objects.get(id=album_id)
     except Album.DoesNotExist:
         logger.exception("Album %s does not exist", album_id)
-        archive.delete()
```

Comment:
This shouldnt make a difference right?

Reply:
since archive = upload.file will just point to the correct object? Or am i overlooking something

Reply:
Should be equivalent yes, I just got confused by the mixed use and want to avoid any risk that it might not be equivalent if somehow the tmp upload object gets modified

Reply:
Fair enough